### DISCUSSION TITLE
v3.20.0

### DISCUSSION THREAD

**Discussion by mehalter** [AstroNvim/AstroNvim#1978] [Announcements]
### Features

* **status:** add provider for `file_encoding` ([cb5bf2a](https://github.com/AstroNvim/AstroNvim/commit/cb5bf2a7dc742c86a1fc545d00691bc14df1d3d5))

**Full Changelog**: https://github.com/AstroNvim/AstroNvim/compare/v3.19.1...v3.20.0

<hr /><em>This discussion was created from the release <a href='https://github.com/AstroNvim/AstroNvim/releases/tag/v3.20.0'>v3.20.0</a>.</em>

**Comment by wqk151** [comment]
Is this wrong, this should be file format ,not file encoding.

**Reply by mehalter** [reply]
No, this is correct, it is providing the file encoding

**Reply by mehalter** [reply]
Oh wait I see what you mean now! This is a typo :o I'm so sorry! I will release a patch fix now to fix

**Reply by mehalter** [reply]
Thanks for catching this @wqk151 I just released v3.21.0 that fixes the `file_encoding` function and just because I wrote it on accident added the `file_format` provider as well
